pas/PAS-44-Onboard-paras-slots.md

```diff
@@ -0,0 +1,41 @@
+---
+PAS: 0044
+title: Onboarding Parachains - Slots
+status: Proposed
+author: Alejandro (@al3mart)
+created: 27-02-2024
+---
+
+## Summary
+Means to onboard a parachain to Paseo via **leases and slot assingment**.
+This PAS is to be deprecated once parachain scheduling via coretime is deployed in Paseo.
+
+## Abstract
+In this document users can find the steps to follow to register their parachain in Paseo.
+- Maintainers of a live parachain in Polkadot or Kusama will be granted a slot in Paseo with no expiration date.
+- Maintainers of parachains in development, or those that have not yet secured a slot in Kusama or Polkadot, will be assigned a lease for a shorter period of time.
```

Comment:
I think we should make the decision about what is a "shorter period of time". 1 week / 2 weeks ? 
We need to take into account the amount of available validators.

CC: @itsbirdo @educlerici-zondax